src/redux/categories/categories.js

```diff
@@ -0,0 +1,13 @@
+export const checkStatus = () => ({ type: 'CHECK_STATUS' });
+const initialState = [];
+
+const checkReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case 'CHECK_STATUS':
+      return 'Under construction';
+    default:
+      return state;
+  }
+};
+
+export default checkReducer;
```

Comment:
- [ ]  In this milestone you have to configure your Redux Store, write actions and reducer for adding and removing books however, I'm getting this error message in the console when I'm trying to add or remove a book, Kindly fix this and make sure the user can add and a new book remove it from the list without any error 😜 👍🏻 

![21bad](https://user-images.githubusercontent.com/79058364/174737043-eec985e1-d27d-4cbf-9f2b-a4adeef1c70c.png)